libraries/textcomposer/impl/src/main/kotlin/io/element/android/libraries/textcomposer/TextComposer.kt

```diff
@@ -668,7 +668,11 @@ internal fun TextComposerFormattingPreview() = ElementPreview {
         )
     }, {
         ATextComposer(
-            RichTextEditorState("A message\nWith several lines\nTo preview larger textfields and long lines with overflow", initialFocus = false),
+            RichTextEditorState(
+                initialHtml = "A message\nWith several lines\nTo preview larger textfields and long lines with overflow",
+                initialMarkdown = "A message\nWith several lines\nTo preview larger textfields and long lines with overflow",
+                initialFocus = false
+            ),
```

Comment:
Use the new fun `aRichTextEditorState()` here too? This place has been forgotten :).